plan/physical_plan_builder.go

```diff
@@ -668,6 +689,7 @@ func (p *Aggregation) convert2PhysicalPlanStream(prop *requiredProperty) (*physi
 		AggFuncs:     p.AggFuncs,
 		GroupByItems: p.GroupByItems,
 	}
+	agg.correlated = p.IsCorrelated()
```

Comment:
if don't add this. the value of correlated of one PhysicalAggregation will always be false though the correlated of its logical plan can be true.